docs/configuration.md

```diff
@@ -0,0 +1,140 @@
+
+There we have 1 RSK Node, 1 Lumino Explorer, 1 Lumino Hub and several Notifiers.
+All these are API's running somewhere, so we need to setup those to let the wallet
+to access them. 
+
+The file that let us configure this is [this one.](../app/scripts/controllers/rif/configuration/index.js)
+There we will see that is a JS file with some functions inside, basically what we need
+to change is the function `getInitialConfigStructure` that makes the initial setup.
+This function looks similar to this:
+
+```js
+  getInitialConfigStructure (chainId) {
+    // ... some other code
+      case global.networks.reg: // RSK Regtest
+        return {
+          lumino: {
+            explorer: {
+              endpoint: '',
+            },
+          },
+          notifier: {
+            availableNodes: [],
+          },
+          rns: {
+            contracts: {
+              rns: '',
+              publicResolver: '',
+              multiChainResolver: '',
+              rif: '',
+              fifsAddrRegistrar: '',
+              rskOwner: '',
+            },
+          },
+        };
+    }
+  }
+```
+**Note: We only show the important piece of code here.**
+
+As you can see we have a structure like this:
+
+```json5
+{
+  lumino: {
+    explorer: {
+      endpoint: '',
+    },
+  },
+  notifier: {
+    availableNodes: [],
+  },
+  rns: {
+    contracts: {
+      rns: '',
+      publicResolver: '',
+      multiChainResolver: '',
+      rif: '',
+      fifsAddrRegistrar: '',
+      rskOwner: '',
+    },
+  },
+}
+```
+
+Now we need to fill those empty fields, here is some reference about what
+those fields are:
+
+```json5
+{
+  lumino: {
+    explorer: {
+      endpoint: 'Here you need to put the endpoint url where the Lumino Explorer API is running, for example http://localhost:8080/api/v1',
+    },
+  },
+  notifier: {
+    availableNodes: [
+      // This is an array of notifier endpoints, you need at least 3.
+      'The endpoint for the notifier 1 (where the API is running), for example: http://localhost:8081/',
+      'The endpoint for the notifier 2 (where the API is running), for example: http://localhost:8082/',
+      'The endpoint for the notifier 3 (where the API is running), for example: http://localhost:8083/'
+    ],
+  },
+  rns: {
+    contracts: {
+      // All these values can be retrieved from the RNS Suite deployment summary.
+      rns: 'The RNS Contract Address',
+      publicResolver: 'The Public Resolver Contract Address',
+      multiChainResolver: 'The Multi-chain Resolver Contract Address',
+      rif: 'The RIF Token Address',
+      fifsAddrRegistrar: 'The FIFSAddrRegistrar Contract Address',
+      rskOwner: 'The RSK Owner Contract Address',
+    },
+  },
+}
+```
+
+As you can see we need to setup some endpoints (for Lumino Explorer and Notifiers) and then the RNS Suite Contract Addresses.
+Lumino Hub doesn't need to be here since Lumino Explorer provide that for us in runtime so we are ok.
```

Comment:
```suggestion
Lumino Hub data doesn't need to be included here since the Lumino Explorer provides that for us in runtime, so we're OK in that regard.
```